File-Formats/PDF/Working-with-Security.md

```diff
@@ -1336,3 +1336,90 @@ Returns user password</td><td>
 Returns owner Password (owner password is same as the user password; it allows full permission to users).</td></tr>
 </tbody>
 </table>
+
+## Troubleshooting
+
+ If the document is protected with owner password only, it seems that it can be open without entering password. In that case, can the Syncfusion library remove/change restrictions? And how do you open a document using owner password if it?s different from user password?
+
+ Using our Syncfusion library, you can remove or edit permissions in a document restricted by an owner password using our import pages option. If you provide the correct owner password, you can perform these actions within the same document.
```

Comment:
Using our Syncfusion library, remove or edit permissions in a document restricted by an owner password using our import pages option. If you provide the correct owner password, you can perform these actions within the same document.